hubble-relay/cmd/serve/serve.go

```diff
@@ -51,6 +53,9 @@ func New() *cobra.Command {
 	cmd.Flags().BoolVarP(
 		&f.debug, "debug", "D", false, "Run in debug mode",
 	)
+	cmd.Flags().BoolVar(
```

Comment:
What do think about adding this flag in helm chart https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/charts/hubble-relay/templates/deployment.yaml#L39?

Reply:
Given that there is already such an option for the cilium agent, I'd say that it'd be good to have for hubble-relay as well, yes.